docs/5.community/7.changelog.md

```diff
@@ -6,7 +6,7 @@ description: Discover the latest Nuxt 3 updates.
 
 # Releases
 
-Discover the latest Nuxt updates.
+[Discover the latest Nuxt updates.](https://nuxt.com/docs/community/changelog)
```

Comment:
This is adding a link to itself. Is there a reason you think this is needed?

Reply:
We don't have source code but a Nuxt redirect is suspected here.

1. Go on this Google search page and click on the third link , https://www.google.com/search?q=nuxt+changelog&rlz=1C1ONGR_frFR1041FR1041&oq=nuxt+changelog&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIICAEQABgWGB4yCAgCEAAYFhgeMgYIAxBFGDwyBggEEEUYPDIGCAUQRRg80gEIMjQ5NmowajeoAgCwAgA&sourceid=chrome&ie=UTF-8